test/ui/index.ts

```diff
@@ -11,12 +11,12 @@ async function main(): Promise<void> {
         const testPath = path.join(__dirname, "command.test.js");
         const exTester = new ExTester();
         // The current version (4.1.1) of vscode-extension-tester doesn't support the newest VSCode version (^1.58.0)
-        await exTester.downloadCode("1.57.0");
+        await exTester.downloadCode("1.59.0");
```

Comment:
I think we should get the version number defined in package.json `"@types/vscode": "1.xx.x"`
`